prober/derp.go

```diff
@@ -196,12 +201,12 @@ func (d *derpProber) probeMapFn(ctx context.Context) error {
 					}
 				}
 
-				if d.bwInterval > 0 && d.bwProbeSize > 0 {
-					n := fmt.Sprintf("derp/%s/%s/%s/bw", region.RegionCode, server.Name, to.Name)
+				if d.bwInterval != 0 && d.bwProbeSize > 0 {
+					n := fmt.Sprintf("derp/%s/%s/%s/bw/%d", region.RegionCode, server.Name, to.Name, d.bwProbeSize)
```

Comment:
I'm including the probe size in the name for clarity. This way, if we wanted to run another probe using different packet sizes, we could look at the resulting metrics separately.

Reply:
Would a label achieve the same thing, or is metric value more useful?

Reply:
Done, I'm calling it `derp_bw_probe_continuous_size_bytes`.